### ISSUE TITLE
Malformed entry in table list

### ISSUE THREAD

**Issue by BorisDomajnko** [keeps/db-preservation-toolkit#400]
Cannot use this table file: tablesA.txt. It was crated with
`... -e list-tables -ef tablesA.txt`

tablesA.txt:
```
HR members.Assignments{Id Employee;Id Department;}
HR members.Departments{Id;Dept Name;}
HR members.HR employees{Id;Employee Name;date Hired /hq;}
HR members.Offices{Id;Office Name ščžäöü;}
```
The command:
`java .. dbptk-app-2.6.0-RC.jar migrate  -i postgresql ..  -e siard-2 .. -etf tablesA.txt `

The error:
```
- table-filter = /home/doma/dbdipview/admin/siard/tablesA.txt
2019-09-27 08:13:39,568 [main] INFO  (c.d.m.m.f.ProgressLoggerObserver) Initializing database
2019-09-27 08:13:39,577 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Closing connection to source database
2019-09-27 08:13:39,586 [main] ERROR (c.d.Main) Fatal error while converting the database (Malformed entry in table list: HR members.Assignments{Id Employee;Id Department;})
com.databasepreservation.model.exception.ModuleException: Malformed entry in table list: HR members.Assignments{Id Employee;Id Department;}
	at com.databasepreservation.modules.siard.out.output.SIARDExportDefault.getModuleSettings(SIARDExportDefault.java:120)
	at com.databasepreservation.model.modules.filters.IdentityFilter.getModuleSettings(IdentityFilter.java:62)
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.migrateDatabaseTo(JDBCImportModule.java:1987)
	at com.databasepreservation.DatabaseMigration.migrate(DatabaseMigration.java:102)
	at com.databasepreservation.Main.runMigration(Main.java:295)
	at com.databasepreservation.Main.internalMain(Main.java:122)
	at com.databasepreservation.Main.main(Main.java:76)
```

**Comment by hmiguim** [comment]
Hi Boris,

Problem fixed, we were not taking into consideration that both schema and table could have spaces in their name.

**Comment by hmiguim** [comment]
Closed at 831af37
